### DISCUSSION TITLE
Drawio source duplicated in build directory

### DISCUSSION THREAD

**Discussion by fabianoriccardi** [LukeCarrier/mkdocs-drawio-exporter#37] [Q&A] [answered]
Hi, when I run `mkdocs build`, in the `site` directory I found the drawio source file as well the `svg` version. Can I disable the copy of the drawio source?

My `mkdocs.yml` is minimal and consist of:

    site_name: My Docs
    plugins:
      - drawio-exporter

**Comment by LukeCarrier** [comment, marked as answer]
Hi @fabianoriccardi,

The plugin doesn’t currently have an option for this and it’s not something I’m likely to add.

You could do this yourself with `find` in your build pipeline:

```console
find site/ -name '*.drawio' -delete
```

**Comment by fabianoriccardi** [comment]
Thanks for the replay. I was just wondering.. why would you keep the source files in the build directory if it isn't linked to the rest of the docs?

**Reply by LukeCarrier** [reply]
Because they’re already copied there by mkdocs, and you might be directly linking to those files to allow users to download them.

mkdocs itself doesn’t follow links and just copies all assets during the build.

**Reply by fabianoriccardi** [reply]
> mkdocs itself doesn’t follow links and just copies all assets during the build.

Got it! Since it mkdocs cannot know that the *.drawio files will be parsed and replaced, it just copies them like any other static asset. Eventually, I will delete the source files with the command you suggested.
